docs/querying/api.md

```diff
@@ -679,6 +679,7 @@ URL query parameters:
 - `rule_name[]=<string>`: only return rules with the given rule name. If the parameter is repeated, rules with any of the provided names are returned. If we've filtered out all the rules of a group, the group is not returned. When the parameter is absent or empty, no filtering is done.
 - `rule_group[]=<string>`: only return rules with the given rule group name. If the parameter is repeated, rules with any of the provided rule group names are returned. When the parameter is absent or empty, no filtering is done.
 - `file[]=<string>`: only return rules with the given filepath. If the parameter is repeated, rules with any of the provided filepaths are returned. When the parameter is absent or empty, no filtering is done.
+- `rule_label_match[]=<string>`: only return rules has the labels that match with label matcher argument. When the parameter is absent, no filtering is done.
```

Comment:
I am curious about parameter name. Why is it `rule_label_match` not just `rule_label`. 

Also, what is format for parameter? Since each label is key value, how do I specify key and value? Is it `:` delimited or? Would be nice to document this here.